puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/lib/rvm.groovy

```diff
@@ -9,8 +9,14 @@ def gemset(name) {
     base_name
 }
 
-def cleanup_rvm(name = '') {
-    withRVM(["rvm gemset delete ${gemset(name)} --force"])
+def configureRVM(ruby) {
```

Comment:
should `ruby` have a default value? (and should that be bumped to something newer than `2.0`?)